src/elementary-tweaks/package.yml

```diff
@@ -0,0 +1,21 @@
+name       : elementary-tweaks
+version    : 1
+release    : 1
+source     :
+    - git|https://github.com/elementary-tweaks/elementary-tweaks.git : 47574c8b64e1d362db5055b82334717515977a73
+license    : GPL-3.0
+component  : desktop.pantheon
+summary    : elementary Tweaks
+description: |
+    elementary Tweaks
```

Comment:
Rewrite as: `Tweaks is a system settings panel for Pantheon desktop that lets you easily and safely customise your desktop's appearance`.